plugins/nvim-lsp/options/servers.nix

```diff
@@ -53,6 +53,7 @@ let
     };
     gopls = {
       languages = "Go";
+      packages = [ gopls ];
```

Comment:
Why did you add this?
When packages is not set, it uses the name, which is fine in this case.
Also, just `gopls` is undefined; it should be `pkgs.gopls`.

Reply:
I don't remember >.>